site/src/App/Code/Code.tsx

```diff
@@ -26,15 +26,21 @@ import { FieldOverlay } from 'braid-src/lib/components/private/FieldOverlay/Fiel
 import { hideFocusRingsClassName } from 'braid-src/lib/components/private/hideFocusRings/hideFocusRings';
 import * as styles from './Code.css';
 
-// @ts-ignore
-import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
-import editorTheme from './editorTheme';
+import { PrismLight as SyntaxHighlighter } from 'react-syntax-highlighter';
+import diff from 'react-syntax-highlighter/dist/esm/languages/prism/diff';
+import jsx from 'react-syntax-highlighter/dist/esm/languages/prism/jsx';
+import tsx from 'react-syntax-highlighter/dist/esm/languages/prism/tsx';
+import { editorTheme } from './editorTheme';
 import { ThemedExample } from '../ThemeSetting';
 import usePlayroomScope from 'braid-src/lib/playroom/useScope';
 import { PlayroomStateProvider } from 'braid-src/lib/playroom/playroomState';
 
 type ReactElementOrString = ReactElement | string;
 
+for (const [name, language] of Object.entries({ diff, jsx, tsx })) {
+  SyntaxHighlighter.registerLanguage(name, language);
+}
```

Comment:
Or perhaps 

```suggestion
(async () => {
  for (const name of ['diff', 'jsx', 'tsx']) {
    SyntaxHighlighter.registerLanguage(
      name,
      await import(
        /* webpackChunkName: "syntax-highlighter-languages" */ `react-syntax-highlighter/dist/esm/languages/prism/${name}`
      ),
    );
  }
})();
```

Reply:
This makes a bundle for every single syntax since webpack can't tell we only want 3 of them. It _does_ work, but means we ship a whole bunch of tiny chunks.